blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_fr.properties

```diff
@@ -0,0 +1,92 @@
+# common
+common.date.duration.format=m[ minutes] s[ secondes]
+common.date.duration.hint.format=M [mois], d [jours], h[h], m[min], s[s]
+common.date.readable.long=DD MMM YYYY h:mma Z
+common.date.readable.short=DD MMM h:mma Z
+common.pager.loading=Chargement...
+common.pager.more=Afficher davantage
+# the emptyState keys assume that you will parse them with a markdown parser
+EmptyState.activity=# \u00c0 vos marques, pr\u00eats, ...\nHmm, on dirait qu'il n'y aucune ex\u00e9cution dans l'historique de construction. Consignez au d\u00e9p\u00f4t _{0}_ ou lancer la construction manuellement.
+EmptyState.artifacts=Il n'y a aucun artefact pour cette ex\u00e9cution.\n\n
+EmptyState.branches=# Cr\u00e9er une branche\nCr\u00e9ez une branche dans le d\u00e9p\u00f4t _{0}_ et Jenkins commencera \u00e0 tester vos changements.\n\nEssayez-vous et devenez le h\u00e9ro de votre \u00e9quipe.
+EmptyState.branches.notSupported=# Les branches ne sont pas prises en charge.\nLes contructions de branches ne fonctionnent seulement avec les t\u00e2ches du type _Multibranch Pipeline_. C'est l'une des nombreuses raisons pour passer aux pipelines Jenkins.\n\n[Apprendre davantage](https://jenkins.io/doc/book/pipeline-as-code/)
+EmptyState.changes=Il n'y a aucun changement pour cette ex\u00e9cution.\n\n
+EmptyState.pr.notSupported=# Les demandes de tirage ne sont pas prises en charge.\nLes contructions de demandes de tirage valides ne fonctionnent seulement avec les t\u00e2ches du type _Multibranch Pipeline_. C'est l'une des nombreuses raisons pour passer aux pipelines Jenkins.\n\n[Apprendre davantage](https://jenkins.io/doc/book/pipeline-as-code/)
+EmptyState.pr=# Je pousse, tu tires\nLorsqu'une demande de tirage est ouverte pour le d\u00e9p\u00f4t _{0}_, Jenkins la testera et rapportera l'\u00e9tat de vos changements sur la demande de tirage sur GitHub.
+EmptyState.tests=Il n'y a aucune ex\u00e9cution de tests pour cette construction.\n\n
+Enable=Activer
+# home page / dashboard
+home.header.button.createpipeline=Nouveau Pipeline
+home.header.dashboard=Tableau de bord
+home.pipelineslist.header.branches=Branches
+home.pipelineslist.header.health=Sant\u00e9
+home.pipelineslist.header.name=Nom
+home.pipelineslist.header.pullrequests=Demandes de tirage
```

Comment:
it sounds weird to me to translate "pull-request". I prefer such technical terms to be used in plain english.